examples/shopify/pages/[[...path]].tsx

```diff
@@ -19,7 +20,7 @@ export async function getStaticProps(
   if (!('props' in makeswiftResult)) return makeswiftResult
 
   const products = await getProducts()
-  const product = await getProduct()
+  const product = (await getProduct()) ?? DEFAULT_PRODUCT
```

Comment:
In catchall routes and /makeswift i think it makes sense to use the default product here.

In the /product dynamic route if getProduct is null then we 404 as expected

Reply:
https://github.com/makeswift/makeswift/pull/166#discussion_r970869479